cucumber/authenticators/features/authn_config.feature

```diff
@@ -0,0 +1,86 @@
+Feature: Authenticator configuration
+
+  Background:
+    Given I have user "no-authn-access"
+    And I have user "authn-reader"
+    And I have user "authn-writer"
+    And a policy:
+    """
+    - !policy
+      id: conjur/authn-ldap/test
+      body:
+      - !webservice
+
+      - !webservice
+        id: status
+
+      - !group readers
+
+      - !permit
+        role: !group readers
+        privileges: [ read ]
+        resource: !webservice
+
+      - !group writers
+
+      - !permit
+        role: !group writers
+        privileges: [ update ]
+        resource: !webservice
+
+    - !grant
+      role: !group conjur/authn-ldap/test/readers
+      member: !user authn-reader
+
+    - !grant
+      role: !group conjur/authn-ldap/test/writers
+      member: !user authn-writer
+    """
+
+  Scenario: Authenticator is successfully configured
+    When I login as "authn-writer"
+    And I successfully PATCH "/authn-ldap/test/cucumber" with body:
+    """
+    enabled=true
+    """
+    Then the HTTP response status code is 204
+    And authenticator "cucumber:webservice:conjur/authn-ldap/test" is enabled
+
+    When I successfully PATCH "/authn-ldap/test/cucumber" with body:
+    """
+    enabled=false
+    """
+    Then the HTTP response status code is 204
+    And authenticator "cucumber:webservice:conjur/authn-ldap/test" is disabled
+
+  Scenario: Authenticator does not exist
```

Comment:
in this case it is not that the authenticator does not exist, rather than the account does not exist. Please change this test to `Account does not exist` and add another test for `Authenticator does not exist`. Both tests will return the same response but we can verify that we write the correct error to the log. 

please see [this example](https://github.com/cyberark/conjur/blob/aac94ad69d2d13e07ac9a90e5e9288e01121139b/cucumber/authenticators/features/authn_oidc.feature#L135) so see how we can check the logs.

Reply:
Right you are, good catch. Fixed this test and added a separate test for non-existent webservice.